pkg/virtual/workspaces/registry/rest.go

```diff
@@ -599,16 +599,19 @@ func (s *REST) Create(ctx context.Context, obj runtime.Object, createValidation
 		return nil, kerrors.NewInvalid(tenancyv1beta1.SchemeGroupVersion.WithKind("Workspace").GroupKind(), obj.GetObjectKind().GroupVersionKind().String(), []*field.Error{})
 	}
 
+	if workspace.Spec.Type.Name == "" && workspace.Spec.Type.Path == "" {
+		workspace.Spec.Type.Name = "Universal"
+		workspace.Spec.Type.Path = "root"
+	}
```

Comment:
no, this is not a static default anymore. It depends on the parent workspace, which has a default type for children.

Reply:
I know, but this would probably have to be done in a followup PR, when we completely remove the need for a SAR here, after changing the CTWexists admission plugin to use the owner annotation content.